### ISSUE TITLE
Missing php-inline file?

### ISSUE THREAD

**Issue by mrjoelkemp** [ajaxorg/ace#2412]
Using the latest, minified build of ace with 

`editor.getSession().setMode({path: 'ace/mode/php', inline: true});`

and seeing a failed script load for `snippets/php-inline.js`. I don't see a file with that name in the snippets directory either.

Tried finding an existing issue to no avail.

Any help is appreciated.

**Comment by nightwing** [comment]
This file is not needed for editor to work, i will look why ace tries to load it.

**Comment by AubreyHewes** [comment]
@nightwing 

Hi, I know this is a _**very**_ old issue, though it is **still open** and still seems to be a **valid issue**  ... 

ACE version 1.3.0
Using via: CDN: (cdnjs) https://cdnjs.cloudflare.com/ajax/libs/ace/1.3.0/ace.js

````javascript
const editor = ace.edit('elementId');

editor.setOptions({ enableSnippets: true });

editor.getSession().setMode({ path: 'ace/mode/php', inline: true });

````

Above tries to load [`snippets/php-inline.js`](https://cdnjs.cloudflare.com/ajax/libs/ace/1.3.0/snippets/php-inline.js) which does not exist. 

Thanks! 

If I have the time I may try to pr a fix this #2412 and possibly other related php-inline caveats #3170

**Comment by adrianbj** [comment]
@nightwing - any chance of finally getting this one sorted? I am still duplicating the php.js to php-inline.js and replacing the references to php inside that file to php-inline.js. It works, but it would be nice not to have to. Thanks!
